command/command.go

```diff
@@ -26,6 +26,7 @@ var nocontext = context.Background()
 // Command parses the command line arguments and then executes a subcommand program.
 func Command() {
 	app := kingpin.New("drone", "drone aws runner")
+	PrintAppDetails(app)
```

Comment:
Add a function to print the program details, instead of including it in the Command function.